tests/sentry/monitors/test_tasks.py

```diff
@@ -445,6 +454,7 @@ def test_timeout_exception_handling(self, logger):
             environment=self.environment,
             last_checkin=next_checkin_ts,
             next_checkin=next_checkin_ts + timedelta(hours=24),
+            next_checkin_latest=next_checkin_ts + timedelta(hours=24),
```

Comment:
Do we not have any tests that have a margin?

Reply:
just the `test_missing_checkin` which does not have a margin. i'll add a `test_missing_checkin_with_margin` test